### ISSUE TITLE
500 - Internal error

### ISSUE THREAD

**Issue by vmcosta** [asm-helpful/helpful-web#346]
Hi Guys, 

It seams that I've broke Helpful ;) after adding a custom domain, now all I get is a "500 - Internal error", please advise on what to do.

Thanks

**Comment by chrislloyd** [comment]
Hi @vmcosta can you provide any more details? An account name would be awesome so I can investigate. If you'd prefer it to be private, feel free to email me at christopher.lloyd@gmail.com.

**Comment by vmcosta** [comment]
Thanks @chrislloyd, 

The issue I'm having is on the account named "weaverthemes" somehow I believe this is related to the http:// I've placed in the custom domain, I do believe now that I only needed to add the name following the .tld

Any help is really appreciated.

Thanks

**Comment by chrislloyd** [comment]
Ah cool, I've been seeing weaverthemes in the errors. Looking into it.

**Comment by vmcosta** [comment]
Thanks @chrislloyd.

**Comment by lachlanjc** [comment]
@chrislloyd did you find the error?

**Comment by vmcosta** [comment]
@lachlanjc

 No, it is the same, I still get a 500 - Internal errors. Since I could not wait any longer I've moved on to a different app.

Best
